packages/drivers/odsp-driver-definitions/src/resolvedUrl.ts

```diff
@@ -143,4 +143,15 @@ export interface IOdspResolvedUrl extends IResolvedUrl, IOdspUrlParts {
 	shareLinkInfo?: ShareLinkInfoType;
 
 	isClpCompliantApp?: boolean;
+
+	/**
+	 * Context for given resolved URL
+	 */
+	context?: string;
```

Comment:
Should these properties be `readonly`?

Reply:
they shouldn't becuase we want to be able to set them in createFile.ts